### PR TITLE
CRBN compatibility & Improved planck keymap

### PR DESCRIPTION
**Pull request by ReFil**
The planck/CRBN keymap is the same as the default QMK keymap
Sorry my code is such a mess :)

### CODE REVIEW

File: app/boards/arm/planck/planck_rev6.keymap [hunk 1]
```diff
@@ -12,11 +12,35 @@
...
+       &trans &trans &trans &trans &mo 3 &trans &trans &trans &cp M_NXT &cp M_VOLD &cp M_VOLU &cp M_PLY
+    >;
+    };
+		control { bindings = <
+       &reset &bootloader &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+			 &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+			 &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+       &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+    >;
+    };
```
Comment (Nicell) [inline app/boards/arm/planck/planck_rev6.keymap:43]: Could you clean up the formatting of this a bit, please? Looks like a mix of tabs and spaces messed it up a bit.

File: app/boards/shields/crbn/crbn.keymap [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#include <behaviors.dtsi>
```
Comment (Nicell) [inline app/boards/shields/crbn/crbn.keymap:1]: Missing a license header here

File: app/boards/shields/crbn/crbn.overlay [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          , <&pro_micro_a 1 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+          ;
+  };
+
+
+
+	bt_unpair_combo: bt_unpair_combo {
+		compatible = "zmk,bt-unpair-combo";
+		key-positions = <0 53>;
+	};
```
Comment (Nicell) [inline app/boards/shields/crbn/crbn.overlay:48]: This definition is deprecated and should be removed.

File: app/boards/shields/crbn/crbn.keymap [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+       &trans &trans &trans &trans &mo 3 &trans &trans &trans &cp M_NEXT &cp M_VOLD &cp M_VOLU &cp M_PLAY
+    >;
+    };
+		control { bindings = <
+       &reset &bootloader &bt BT_CLR &bt BT_PRV &bt BT_NXT &trans &trans &trans &trans &trans &trans &trans
+			 &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+			 &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+       &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans &trans
+    >;
+    };
```
Comment (Nicell) [inline app/boards/shields/crbn/crbn.keymap:44]: Same here on the formatting if you could 😄

**Review by Nicell — CHANGES_REQUESTED** [review]
Thanks for your work on this shield! I added some initial thoughts, and then there's a couple files that still need to be added/updated:

[`.github/workflows/build.yml`](https://github.com/zmkfirmware/zmk/blob/main/.github/workflows/build.yml): Add your board to the shield list, so that it's build by GitHub Actions.

[`docs/docs/hardware.md`](https://github.com/zmkfirmware/zmk/blob/main/docs/docs/hardware.md): You should add the CRBN to this list

[`docs/static/setup.ps1`](https://github.com/zmkfirmware/zmk/blob/main/docs/static/setup.ps1) and [`docs/static/setup.sh`](https://github.com/zmkfirmware/zmk/blob/main/docs/static/setup.sh): CRBN should be added as a non-split option for both files.

File: app/boards/shields/crbn/crbn.keymap (hunk 4 above)
Comment (ReFil) [reply app/boards/shields/crbn/crbn.keymap]: I think it's all sorted now

**Review by Nicell — APPROVED** [review]
Thanks!

**Comment by ReFil** [comment]
I've made such a mes of this I'll start over

**Pull request closed without merging**